project.clj

```diff
@@ -1,9 +1,17 @@
+(def clj-version "1.7.0")
+(def ks-version "1.3.0")
+(def tk-version "1.3.1")
+
 (defproject puppetlabs/ring-middleware "0.2.3-SNAPSHOT"
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/tools.logging "0.2.6"]
-                 [ring "1.3.0" :exclusions [org.clojure/java.classpath]]
-                 [puppetlabs/ssl-utils "0.8.0" :exclusions [clj-time]]
-                 [puppetlabs/http-client "0.4.4" :exclusions [prismatic/schema commons-io clj-time]]]
+  :dependencies [[cheshire "5.6.1"]
+                 [org.clojure/clojure ~clj-version]
+                 [org.clojure/tools.logging "0.3.1"]
+                 [prismatic/schema "1.1.0"]
+                 [puppetlabs/http-client "0.5.0" :exclusions [prismatic/schema commons-io clj-time ]]
+                 [puppetlabs/kitchensink ~ks-version :exclusions [clj-time]]
+                 [puppetlabs/ssl-utils "0.8.1" :exclusions [clj-time]]
+                 [ring "1.4.0" :exclusions [org.clojure/java.classpath]]
+                 [slingshot "0.12.2"]]
```

Comment:
Oh, that's a good write up. Thanks! I'll need to update the associated PRs for the downstream repos as well. Updated in 6b84b9b4aa4a7b0d0f7780da713de175f0da9b00